src/core/ext/client_config/lb_policy_factory.h

```diff
@@ -82,9 +87,12 @@ void grpc_lb_addresses_destroy(grpc_lb_addresses *addresses,
                                void (*user_data_destroy)(void *));
 
 /** Arguments passed to LB policies. */
+/* TODO(roth, ctiller): Consider replacing this struct with
+   grpc_channel_args.  See comment in resolver_result.h for details. */
 typedef struct grpc_lb_policy_args {
   grpc_lb_addresses *addresses;
   grpc_client_channel_factory *client_channel_factory;
+  grpc_channel_args *additional_args;
```

Comment:
I added a comment explaining that it's for passing in implementation-specific parameters to the LB policy.  But since we don't actually have any use-cases for this yet, I don't have an example to add.